app/controllers/api/v1/diaper_calculators_controller.rb

```diff
@@ -9,6 +9,10 @@ class DiaperCalculatorsController < ApplicationController
       ].freeze
 
       def create
+        res =
```

Comment:
Not beautiful enough, maybe better like so: 
```suggestion
        result = diapers_service_handler(childs_birthday).calculate!
        ...
        
        private
        
        def diapers_service_handler
          @diapers_service_handler ||= Calculators::DiapersService.new
        end
        
        def childs_birthday # because you call it twice in `#create` method
          params[:childs_birthday]
        end
```

Reply:
Try to refactor your code similar to my suggestion, keep your action methods thin